release-tools/check-crate-version/toml-diff/src/test.rs

```diff
@@ -0,0 +1,245 @@
+use super::{TomlChange, TomlDiff};
```

Comment:
These tests look good, but can you add a control case where there is no diff

Reply:
Added